src/contracts/atlas/Escrow.sol

```diff
@@ -92,12 +102,13 @@ abstract contract Escrow is AtlETH {
     {
         bool _success;
         bytes memory _data;
+        uint256 _gasLimit = userOp.gas > gasleft() ? gasleft() : userOp.gas;
```

Comment:
Shouldn't there be an offset to allow graceful returns if we run out of gas? plus the issue about the call forcing 63/64?  We may want to do `gasleft() * 63 / 64 - gracefulReturnOffset', where `gracefulReturnOffset` is enough to store what we need and return the correct atlas-formatted error. (50k?)

Reply:
Yeah, good call, will add that

Reply:
PR here: https://github.com/FastLane-Labs/atlas/pull/446

Will merge that into Atlas v1.1 and then update this Arbitrum version with the changes if PR is accepted.